placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -145,10 +171,39 @@ public void onItemClick(final AdapterView<?> parent, final View view, final int
                 }
             }
         });
-
+        if(clearEnabled)
```

Comment:
Nit: prefer to use `{` here instead of shorthand. we try to follow google's java style guide

Reply:
Fixed